apps/ios/Shared/Model/ChatModel.swift

```diff
@@ -428,19 +436,17 @@ final class ChatModel: ObservableObject {
 
     private func _upsertChatItem(_ cInfo: ChatInfo, _ cItem: ChatItem) -> Bool {
         if let i = getChatItemIndex(cItem) {
-            withConditionalAnimation {
-                _updateChatItem(at: i, with: cItem)
-            }
```

Comment:
Two instances of `withConditionalAnimation ` removed.
- They haven't had effect since moving to reverse list
- They where interfering with chat item view update, when item below it causes the change